cmd/validator/flags/flags.go

```diff
@@ -230,6 +230,12 @@ var (
 		Name:  "force-exit",
 		Usage: "Exit without displaying the confirmation prompt",
 	}
+	VoluntaryExitJSONOutputPath = &cli.StringFlag{
```

Comment:
you need to add this to the usage.go file I believe otherwise it won't show up in `help`

Reply:
It only applies to subcommands in prysmctl. I put this flag everywhere that ForceExit appears

Reply:
![Screenshot from 2023-04-17 11-54-46](https://user-images.githubusercontent.com/7246818/232556389-e851875a-ef99-4679-b366-778d47254b25.png)